dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -60,10 +60,21 @@ def setUpClass(cls):
         cls.aws_dataset = pd.read_csv(cls.input_file_path)
         profiler_options = ProfilerOptions()
         profiler_options.set({"data_labeler.is_enabled": False})
+
+        profiler_options_hll = ProfilerOptions()
```

Comment:
this is going to be really expensive to recompute all of aws again with different options. I wonder if we can move these tests elsewhere or create tests that focus directly on the hll work.

Reply:
moved row stat tests to `TestStructuredProfilerRowStatistics` class below